src/community/china.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Using Flutter in China
+description: Where to a version of flutter.io localized to Mandarin.
+toc: false
+---
+
+The Flutter community has made a Mandarin version of the flutter.io
+website available at [https://flutter-io.cn](https://flutter-io.cn.).
```

Comment:
- [ ] There is an **error in the link**: note the extraneous period at the end of the URL.
- [ ] IMHO we should drop the `https://` from the link text.

Reply:
I do now.  ;)